src/_locales/en/messages.json

```diff
@@ -274,11 +274,16 @@
     "description": "Label for hidden results"
   },
 
-  "notApplicablelabel": {
+  "notApplicableLabel": {
     "message": "Not Applicable",
     "description": "Label for not applicable results"
   },
 
+  "notApplicableDescRule": {
```

Comment:
It look like this is another message that was created for this branch, but is not being used.